example.cfg

```diff
@@ -3,24 +3,25 @@ ModelInterface = /people/link593/wrk/ModelInterface-baseX/ModelInterface.jar
 DBXMLlib = /people/link593/lib
 inputdir = ./input-data
 rgnconfig = rgn32
+modtable = module-table.csv
```

Comment:
No longer our approach.

Reply:
Thanks, I'd missed this one and the one below.  You should be able to delete these via GitHub's new suggestion feature.